Remora.Discord.Interactivity/README.md

```diff
@@ -95,6 +95,30 @@ public Task<Result> OnMenuSelectionAsync(IReadOnlyList<MyArbitraryType> values)
 The raw values in question are taken from the select menu options of the 
 component, and can be any parseable data.
 
+When using `user`, `role`, `mentionable` and `channel` select menus, no values are returned.
```

Comment:
I'd prefer it if we implemented support for taking the IUser/IRole/etc arguments as a list parameter instead of this caveat. It might already be implemented "for free" since the system uses the resolved field to get local cached copies of the instances - if not, the new resolved field might need to be wired into the caching system in this PR.

Reply:
Indeed, I've implemented it such that the IDs of any resolved data are passed as command parameters, and let the built-in resolvers for `IChannel` etc. handle if from there.